lib/betagouvbot/format_mail.rb

```diff
@@ -26,6 +26,10 @@ def call(context)
       end
     end
 
+    def format_recipients(context)
```

Comment:
I'd prefer to leave this method private, and change `.call` return as suggested ↑ .